include/seeds.proposals.hpp

```diff
@@ -89,6 +89,12 @@ CONTRACT proposals : public contract {
 
       ACTION initnumprop();
 
+      ACTION delgatetrust(name delegator, name delegatee, name scope);
+
+      ACTION mimicvote(name delegatee, name scope, uint64_t proposal_id, double percentage_used, name option, uint64_t chunksize);
+
+      ACTION cncldeltrst(name delegator, name scope);
```

Comment:
delegate()
undelegate()

for these functions?

Reply:
I changed the names of the functions